src/strawberry_sqlalchemy_mapper/field.py

```diff
@@ -0,0 +1,731 @@
+    name: str | None = None,
+    field_name: str | None = None,
+    is_subscription: bool = False,
+    description: str | None = None,
+    permission_classes: list[type[BasePermission]] | None = None,
+    deprecation_reason: str | None = None,
+    default: Any = dataclasses.MISSING,
+    default_factory: Callable[..., object] | object = dataclasses.MISSING,
+    metadata: Mapping[Any, Any] | None = None,
+    directives: Sequence[object] | None = (),
+    graphql_type: Any | None = None,
+    extensions: Sequence[FieldExtension] = (),
+    sessionmaker: _SessionMaker | None = None,
+    # This init parameter is used by pyright to determine whether this field
+    # is added in the constructor or not. It is not used to change
+    # any behavior at the moment.
+    init: Literal[True, False, None] = None,
+) -> Any:
+    """Annotate a method or property as a SQLAlchemy GraphQL field.
+
+    Examples
+    --------
+        It can be used both as decorator and as a normal function:
+
+        >>> @strawberry.django.type
+        >>> class X:
+        ...     field_abc: str = strawberry.django.field(description="ABC")
+        ...     @strawberry.django.field(description="ABC")
+        ...
+        ...     def field_with_resolver(self) -> str:
+        ...         return "abc"
+
+    """
+    f = field_cls(
+        python_name=None,
+        graphql_name=name,
+        type_annotation=StrawberryAnnotation.from_annotation(graphql_type),
+        description=description,
+        is_subscription=is_subscription,
+        permission_classes=permission_classes or [],
+        deprecation_reason=deprecation_reason,
+        default=default,
+        default_factory=default_factory,
+        metadata=metadata,
+        directives=directives or (),
+        extensions=cast(List[FieldExtension], extensions),
+        sessionmaker=sessionmaker,
+    )
+
+    if resolver:
+        return f(resolver)
+
+    return f
+
+
+def node(
+    *,
+    name: str | None = None,
+    field_name: str | None = None,
+    is_subscription: bool = False,
+    description: str | None = None,
+    permission_classes: list[type[BasePermission]] | None = None,
+    deprecation_reason: str | None = None,
+    default: Any = dataclasses.MISSING,
+    default_factory: Callable[..., object] | object = dataclasses.MISSING,
+    metadata: Mapping[Any, Any] | None = None,
+    directives: Sequence[object] | None = (),
+    graphql_type: Any | None = None,
+    extensions: Sequence[FieldExtension] = (),
+    sessionmaker: _SessionMaker | None = None,
+    # This init parameter is used by pyright to determine whether this field
+    # is added in the constructor or not. It is not used to change
+    # any behavior at the moment.
+    init: Literal[True, False, None] = None,
+) -> Any:
+    """Annotate a property to create a relay query field.
+
+    Examples
+    --------
+        Annotating something like this:
+
+        >>> @strawberry.type
+        >>> class X:
```

Comment:
nit: I think this documentation would be a bit clearer if this used `class Query`